### ISSUE TITLE
GPKG Vector Features and Vector Tiles

### ISSUE THREAD

**Issue by techmavengeospatial** [sqlitecloud/docs#31]
Have you thought of including support for Geospatial/GIS Map Data?

Specifically GeoPackage which also uses SQLite

- Vector Features - (geometry blob similar to WKB well known binary and regular attribute fields/data)
- Vector Tiles (tile_data blob GZ PBF Protocol Buffer Format)
- Both of these require support tables and metadata (GPKG_Contents, and others)

Spatialite library can read GeoPackage  there is also SPL.JS WASM Web Assembly Library

**Comment by marcobambini** [comment]
@techmavengeospatial yes, sure, we'll be ready to announce our SQLite extensions support within a few weeks
